src/pages/EditMonitor.vue

```diff
@@ -804,11 +814,22 @@ message HealthCheckResponse {
                         this.monitor.notificationIDList[this.$root.notificationList[i].id] = true;
                     }
                 }
-            } else if (this.isEdit) {
+            } else if (this.isEdit || this.isClone) {
```

Comment:
Perhaps `this.isEdit`, `this.isClone` should be separated for better readability, as there is a check inside `this.isEdit` for `this.isClone`. Yes, some code will be duplicated, but it will make it easier and more convenient to change the logic of `this.isClone` regardless of the logic of `this.isEdit`

Reply:
I would disagree